src/quicklist.c

```diff
@@ -138,41 +149,52 @@ quicklist *quicklistNew(int fill, int compress) {
     return quicklist;
 }
 
-void quicklistRepr(unsigned char *ql)
-{
+void quicklistRepr(unsigned char *ql) {
+    int i=0;
     quicklist *quicklist  = (struct quicklist*) ql;
-
-    printf("count=%ld\n", quicklist->count);
-    printf("len=%ld\n", quicklist->len);
-    printf("fill=%d\n", quicklist->fill);
-    printf("compress=%d\n", quicklist->compress);
-    printf("bookmark_count%d\n", quicklist->bookmark_count);
+    printf("{count : %ld}\n", quicklist->count);
+    printf("{len : %ld}\n", quicklist->len);
+    printf("{fill : %d}\n", quicklist->fill);
+    printf("{compress : %d}\n", quicklist->compress);
+    printf("{bookmark_count : %d}\n", quicklist->bookmark_count);
     quicklistNode* node = quicklist->head;
-    int i=0;
     while(node != NULL) {
-        printf("===========================================\n");
-        printf("============quicklist node(%d)============\n", i++);
-        printf("===========================================\n");
-        printf("container=%d\n", node->container);
+        printf("{quicklist node(%d)\n", i++);
+        printf("{container : %d}\n", node->container);
         if (node->container == QUICKLIST_NODE_CONTAINER_ZIPLIST) {
-            printf("ziplist:\n");
-            ziplistRepr(node->zl);
+            printf("{ ziplist:\n");
+            ziplistRepr(node->entry);
+            printf("}\n");
+
         }
         else if (node->container == QUICKLIST_NODE_CONTAINER_NONE) {
-            printf("entry=%s\n", node->entry);
+            printf("{ entry : %s }\n", node->entry);
         }
-        printf("===========================================\n");
-        printf("===========================================\n");
-        printf("===========================================\n");
+        printf("}\n");
         node = node->next;
+    }
+}
 
+void quicklistReprLight(unsigned char *ql) {
+    int i=0;
+    quicklist *quicklist  = (struct quicklist*) ql;
+    printf("{count : %ld}\n", quicklist->count);
+    printf("{len : %ld}\n", quicklist->len);
+    printf("{fill : %d}\n", quicklist->fill);
+    printf("{compress : %d}\n", quicklist->compress);
+    printf("{bookmark_count : %d}\n", quicklist->bookmark_count);
+    quicklistNode* node = quicklist->head;
+    while(node != NULL) {
+        printf("{quicklist node(%d)\n", i++);
+        printf("{container : %d}\n", node->container);
```

Comment:
let's also print the compression flag, and the size in the light mode.
and let's convert the `container` to string, i.e.
```suggestion
        printf("{container : %s, encoding: %s, size: %zu}\n", node->container==PLAIN? "PLAIN": "PACKED", node->encoding==RAW? "RAW": "LZF", node->sz);
```